lib/reporter.js

```diff
@@ -84,7 +84,8 @@ class CucumberReporter {
         case Cucumber.Status.PENDING:
         case Cucumber.Status.SKIPPED:
         case Cucumber.Status.AMBIGUOUS:
-            e = 'pending'
+            e = 'skipped'
```

Comment:
@just-boris I still don't know why we change the event name here, can't we handle this in the allure reporter only?